src/main/java/io/securecodebox/persistence/defectdojo/service/DefaultImportScanService.java

```diff
@@ -40,7 +40,7 @@
 /*
  * https://defectdojo.security.iteratec.dev/api/v2/oa3/swagger-ui/#operations-tag-import-scan
  */
-class DefaultImportScanService implements ImportScanService {
+public class DefaultImportScanService implements ImportScanService {
```

Comment:
Why public?

Reply:
The class is used in the `VersionedEngagementsStrategy` class in the [persistence-defectdojo hook](https://github.com/secureCodeBox/secureCodeBox/blob/main/hooks/persistence-defectdojo/hook/src/main/java/io/securecodebox/persistence/strategies/VersionedEngagementsStrategy.java#L66). If is stays package-private, we get: 
```
'io.securecodebox.persistence.defectdojo.service.DefaultImportScanService' is not public in 'io.securecodebox.persistence.defectdojo.service'. Cannot be accessed from outside package
```

Reply:
From now on you must use the interface type `ImportScanService` and its factory method `ImportScanService#createDefault(Config)` to create an instance.

This is necesary to decouple the implementation from the interface so we can do refactorings easier in future.